client/src/common/types/entities.ts

```diff
@@ -2,20 +2,33 @@
 
 export type Category = number | string | boolean;
 
+export type ColumnSchema = AnnotationColumn | LayoutColumn | DataFrame;
+
 export interface AnnotationColumn {
```

Comment:
Needs a more descriptive name to differentiate it from an actual Annotation Column.  Suggest adding suffix `Schema`, eg, `AnnotationColumnSchema`

Reply:
Updated in #2375.